Makefile

```diff
@@ -420,13 +420,13 @@ uninstall:
 		/config/overlays/dev | kubectl delete -f -
 
 __controller-gen: __tooling-image
-CONTROLLER_GEN=docker run -v $(shell pwd):/gatekeeper gatekeeper-tooling controller-gen
+CONTROLLER_GEN=docker run --rm -v $(shell pwd):/gatekeeper gatekeeper-tooling controller-gen
 
 __conversion-gen: __tooling-image
-CONVERSION_GEN=docker run -v $(shell pwd):/gatekeeper gatekeeper-tooling conversion-gen
+CONVERSION_GEN=docker run --rm -v $(shell pwd):/gatekeeper gatekeeper-tooling conversion-gen
 
 __tooling-image:
-	docker build . \
+	docker build build/tooling \
 		-t gatekeeper-tooling \
 		-f build/tooling/Dockerfile
```

Comment:
do we still need this?

Reply:
IIRC this dockerized conversion-gen and controller-gen, which helped stabilize output formats and avoided requiring contributors install the binaries manually.

Reply:
sorry i meant `-f build/tooling/Dockerfile` line

Reply:
ah, yeah, not sure that's needed